exampleCourse/questions/exampleCodeOutput/question.html

```diff
@@ -0,0 +1,19 @@
+<pl-question-panel>
```

Comment:
File name should be `examplesCodeOutput` instead of `exampleCodeOutput` (note the `s`).